content/docs/using-github-directly/index.mdx

```diff
@@ -0,0 +1,303 @@
+
+## Self-assessment activities
+
+Self-assessment activities such as online tests or quizzes enliven the learning process; address different learning styles; and provide opportunities to expand upon the course content in the classroom or independently. These can include quizzes; exercises; assignments; questions to consider in a group discussion; practical step-by-step tutorials guiding learners through a task; and much more. Since the technical guidelines for including this kind of material will vary according to their nature, the DARIAH-Campus editorial team will be happy to discuss your requirements as you develop your resource.
+
+## Language and readability
+
+Where writing in English, keep an international audience in mind and avoid idioms and colloquialisms where possible. We are open to submissions in languages other than English. We are committed to inclusiveness, and welcome new additions to the editorial and reviewing team, in order to expand our reach. Please contact us via the DARIAH Helpdesk, if you would like to be a part of this team.
+
+## Openness and Licencing
+
+DARIAH-Campus learning materials are licensed under a Creative Commons CC BY-SA licence. This ensures that they are available for free; can be re-mixed and repurposed; and are easy to combine with other materials and courses. The integrity of the resource is ensured under the ‘Attribution’ requirement, whereby re-users must give appropriate credit, provide a link to the license, and indicate if changes were made.
+
+
+# GitHub Workflow
+
+DARIAH-Campus uses GitHub for its editorial workflow.
+
+Once you start working on your contribution, you should make sure you get into the habit of regularly syncing your local clone with the remote clone and the original repository, as well as committing (saving) your work both to your local clone and your remote server.
+
+## Saving your contribution
+
+This may sound complicated, but is actually quite logical. Once you have done it several times, it will become like second nature to you:
+
+1. **Fetch origin**. Before you start a working session on your computer, make sure you "fetch origin": ![](images/github_desktop_fetch_origin.png) This will update your local fork with possible changes you've made to your "origin", i.e. your fork on GitHub.com. This is important if you're, for instance, working on more than one computer.
+2. **Merge from upstream**. Also, before you start your working session, you should merge from upstream by going to Branch > Merge into current branch > Select upstream/master and then clicking on "Merge upstream/master into master" ![](images/github_merge_from_upstream.png) This will update your local fork with the changes that may have been submitted to the "upstream" DARIAH-CAMPUS, i.e. the original repository owned by DARIAH.
+3. Do your work. On how to set up a new learning resource see USING TEMPLATES. TODO!
```

Comment:
TODO?

Reply:
yeah - I think that was a note to self that got left in the last time round!  Thanks for spotting it, I'll delete it.